<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Config</class>
 <widget class="QWidget" name="Config">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>236</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="General">
     <property name="title">
      <string>General options</string>
     </property>
     <layout class="QVBoxLayout">
      <item>
       <widget class="QCheckBox" name="UseWeight">
        <property name="toolTip">
         <string>Specifies if the meta-engine makes a normal or a weighted sum of the rankings (using the weights defined by each search engine called).</string>
        </property>
        <property name="text">
         <string>Use Engine weight</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="QueryModeOption">
     <property name="toolTip">
      <string>Choose the working mode of the meta engine</string>
     </property>
     <property name="whatsThis">
      <string>The General Option allows the user to choose between different working mode for the &lt;b&gt;Meta Engine&lt;/b&gt; :&lt;br&gt;
&lt;u&gt;&lt;b&gt;1.Single Query Mode:&lt;/b&gt;&lt;/u&gt;
&lt;p&gt; The set of keywords received by the meta engine is used in is totality to create one query.&lt;/p&gt;
&lt;u&gt;&lt;b&gt;2. Combinaison of query terms:&lt;/b&gt;&lt;/u&gt;
&lt;p&gt;The set of keywords received by the meta engine is used to create several combinaison of the keywords. All these combinaisons are used as query. The meta engine retreive the results for each combinaison of keywords and the rank all the results together. &lt;br&gt;
The &lt;b&gt;among&lt;/b&gt; of combinaison is determiner by the PUpper Value witch determine the &quot;p&quot; value for a combinaison Cnp where &quot;n&quot; is the number of keywords retreived by the meta engine&lt;/p&gt;
&lt;u&gt;&lt;b&gt;3. Ranch of combinaisons of the query terms&lt;/b&gt;&lt;/u&gt;
&lt;p&gt;This option enable the meta engine to use a ranch of combinaisons (defined as in point 2). Two Value must be specified for the &quot;p&quot; value, the lower value and the upper value. Combinaisons are then create using as value &quot;p&quot; all the value between the two threshold&lt;/p&gt;</string>
     </property>
     <property name="title">
      <string>Query Mode option</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QComboBox" name="Type">
        <property name="toolTip">
         <string>Define how the query is interpreted.</string>
        </property>
        <property name="whatsThis">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;There are three types of queries:&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Single&lt;/span&gt;: A single query is build using all the keywords.&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;k-Combinations&lt;/span&gt;: All possible queries using different combinations of a fixed keyword number are build.&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Range of k-Combinations&lt;/span&gt;. All possible queries using different combinations of a range of keyword numbers are build.&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <item>
         <property name="text">
          <string>Single</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>k-Combinations</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="PLowerPCLabel">
          <property name="text">
           <string>Min k</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="kMin"/>
        </item>
        <item>
         <spacer name="spacer5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>13</width>
            <height>21</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="PUpperPCLabel">
          <property name="text">
           <string>Max k</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="kMax"/>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QCheckBox" name="PercentValue">
          <property name="toolTip">
           <string>Specifies if the keyword numbers are percentage or absolute values.</string>
          </property>
          <property name="text">
           <string>Use as percentage</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
